claim/test_validations.py

```diff
@@ -259,8 +259,8 @@ def test_validate_patient_category(self):
 
         # Then
         claim1.refresh_from_db()
-        self.assertEquals(len(errors), 1)
-        self.assertEquals(errors[0]['code'], 1)  # claimed rejected because all services are rejected
+        self.assertGreaterEqual(len(errors), 1)
+        self.assertEquals(errors[-1]['code'], 1)  # claimed rejected because all services are rejected
```

Comment:
I don't think using -1 is correct approach in here. Currently it's working as you most likely have 2 elements in errors output, however when we'll have another one it'll fail one again (unless it's sure that rejected reason will be last code, which is still non-intuitive in my opinion).

Reply:
It is sure it will be the last one.